### ISSUE TITLE
Adjusting minimum value

### ISSUE THREAD

**Issue by abdullahabbasi** [bernii/gauge.js#84]
Gauge disappears when i put min value.
http://jsfiddle.net/v8e3sd1b/3/

**Comment by fritz-smh** [comment]
Hi,

Same for me. Here is the traceback in chrome when I set a minValue != 0 : 

```
Exception caught during observer callback: TypeError: Cannot read property 'pct' of undefined
    at b.getColorForPercentage (http://192.168.1.50:50002/widget/gauges/js/gauge.min.js:1:8782)
    at b.getColorForValue (http://192.168.1.50:50002/widget/gauges/js/gauge.min.js:1:9122)
    at b.render (http://192.168.1.50:50002/widget/gauges/js/gauge.min.js:1:9485)
    at b.a.update (http://192.168.1.50:50002/widget/gauges/js/gauge.min.js:1:2321)
    at Object.window.AnimationUpdater.run (http://192.168.1.50:50002/widget/gauges/js/gauge.min.js:1:12999)
    at b.set (http://192.168.1.50:50002/widget/gauges/js/gauge.min.js:1:8290)
    at dmw-gauges-gauge1.Polymer.sensorvalueChanged (http://192.168.1.50:50002/widget/gauges/gauge1.html:92:27)
    at dmw-gauges-gauge1.properties.invokeMethod (http://192.168.1.50:50002/libraries/polymer/polymer.js:9385:12)
    at dmw-gauges-gauge1.properties.notifyPropertyChanges (http://192.168.1.50:50002/libraries/polymer/polymer.js:9374:20)
    at Object.Observer.report_ (http://192.168.1.50:50002/libraries/polymer/polymer.js:4913:24)Observer.report_ @ polymer.js:4916createObject.check_ @ polymer.js:5319callback @ polymer.js:4811
```

Regards,
Fritz

**Comment by JerryLosey** [comment]
Same here.

**Comment by hangy233** [comment]
Hi. Fixed this in pull requrest #107

**Comment by kplindegaard** [comment]
Fixed in c1c72e2190d21c9142fe380753e454be2a9e9307
